app/assets/javascripts/models/ui/reverb_retirement.coffee

```diff
@@ -0,0 +1,19 @@
+#= require util/js.cookies
```

Comment:
This file is not needed.

Reply:
It kind of is now - I removed it from the search_page, so now it's only included upon this object instantiation (which of course happens in the ready event).

Reply:
Remove line 1. The lib should be included earlier right after jquery loads.